barretenberg/sol/src/honk/instance/Add2Honk.sol

```diff
@@ -25,30 +25,6 @@ import {ecMul, ecAdd, ecSub, negateInplace, convertProofPoint} from "../utils.so
 // Field arithmetic libraries
 import {Fr, FrLib} from "../Fr.sol";
 
-struct Proof {
```

Comment:
As proof sizes are fixed in the current form, the proof struct can be shared across instances, hence its deletion. 

The remaining changes in the solidity files are majority formatting changes